Shared/Realm.Shared/RealmList.cs

```diff
@@ -170,42 +80,22 @@ public int Count
 
         #region implementing IList members
 
-        /// <summary>
-        /// Makes a relationship to an item, appending it at the end of the sorted relationship.
-        /// </summary>
-        /// <param name="item">RealmObject being added to the relationship.</param>
```

Comment:
Which details do you find useful? I thought all those comments were fairly generic explanation of how lists work and nothing specific about our API.